### PR TITLE
added image and input component

### PR DESCRIPTION
**Pull request by vivek-guda**

### CODE REVIEW

File: Components/image.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.image-responsive {
+  width: 100%;
+  height: auto;
+}
+
+.image-200 {
+  width: 200px;
+  height: 200px;
+  object-fit: cover;
+}
```
Comment (abhishek17551) [inline Components/image.css:10]: recommend to use relative units.

File: Components/input.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  width: 100%;
+  margin: 0.5rem;
+}
+
+.input input {
+  padding: 0.45rem;
+  outline: none;
+  background-color: var(--blueblack-100);
+  border-radius: 5px;
+  margin-top: 5px;
```
Comment (abhishek17551) [inline Components/input.css:11]: recommend to use relative units.

File: docs.html [hunk 3]
```diff
@@ -526,6 +528,142 @@ <h2 class="font-size-md">Men Premium Jacket</h2>
...
+  
+          <h2 class="font-size-lg">Example of Round Image</h2>
+          <p class="font-size-md">
+            You can add class
+            <span class="font-weight-700 style-classes">image-round</span>
+            to make your image round shaped.
+          </p>
+          <div class="component-display-container">
+            <img src="./assets/images/natureImg.png" alt="" class="image-200 image-round" />
+          </div>
```
Comment (abhishek17551) [inline docs.html:569]: can align the image to center of `component-display-container`

**Review by abhishek17551 — APPROVED** [review]
minor changes. rest all looks okay.

File: Components/input.css (hunk 2 above)
Comment (SHIKHASINGH1506) [reply Components/input.css:11]: Don't use px it won't be responsive, instead use rem or em.

**Comment by SHIKHASINGH1506** [comment]
Try to use relative units, that's the major change in this PR other things are fine.

**Pull request merged**